bluepyopt/ephys/locations.py

```diff
@@ -392,6 +393,73 @@ def instantiate(self, sim=None, icell=None):
         return self.find_icomp(sim, branches)
 
 
+class NrnTrunkSomaDistanceCompLocation(NrnSecSomaDistanceCompLocation):
```

Comment:
We also need a test for this class

Reply:
the test for parent class `TestNrnSecSomaDistanceCompLocation` is also missing, but I cannot make it work, the mock `Cell` object does not have `soma[0]` as parent of first section, and I'm not sure how to fix this. If you have any suggestions, I'll give it a try!

Reply:
test added, thank aurelien!